src/main/app/src/components/ohjaava-haku/OhjaavaHakuContext.ts

```diff
@@ -0,0 +1,29 @@
+import { createContext, useContext } from 'react';
+
+import { QuestionType, Rajain } from '#/src/components/ohjaava-haku/Question';
+
+export type OhjaavaHakuContextType = {
```

Comment:
Mahdollisia suorituskykyongelmia, jos laitetaan näin paljon muuttuvia arvoja kontekstiin, koska kaikki kontekstin lapset renderöidään uudelleen, jos konteksti muuttu. Esim. Zustand voisi auttaa tässä. Muuten hyvä rakenne ja ohjaavan haun tiedot on nyt tähän kapseloitu.